src/content/docs/2/guide/recipe/notifications.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: Notifications
+description: Stay tuned while we work on some delicious recipes.
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+In this recipe we'll take a look at sending a native notification to your user. We'll do this by using the official notification plugin.
+
```

Comment:
Would this page contain a demo? I think having those demos as high-up on the page as possible (above the fold) would be best, but would like your feedback.

Reply:
Yep this page will have a demo, it used to have one actually but you wanted me to split that component up into its own PR and it hasn't been merged yet 😅 But yes I imagine it being as far up as possible as well, either at the very top or below the description just above the Setup title